src/main/java/Duke.java

```diff
@@ -3,53 +3,60 @@
 
 public class Duke {
     public static void main(String[] args) {
-        //Implementing Increment Level-0: Initial skeleton of Duke
+        //Starting the bot
         System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
-        Task List[] = new Task[100];
+        Task List[] = new Task[100];  //Creating a list of tasks
         int listSize = 0;
-        while(true){
 
-            String line;
+        while(true) {
```

Comment:
Should leave a space between condition and while
while (condition) {
    statements;
}

Reply:
Similar to what your peers have mentioned, please check and ensure that there are sufficient spaces to meet the coding standard, especially for control flow statements.